docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -236,6 +236,7 @@ All fields in the ``DOCUMENTATION`` block are lower-case. All fields are require
   * Details of any important information that doesn't fit in one of the above sections.
   * For example, whether ``check_mode`` is or is not supported.
 
+.. _module_documents_linking:
 
 Linking and other format macros within module documentation
```

Comment:
Not for this PR, but this needs to be changed a bit, since this isn't about modules, but modules **and plugins**.